specutils 1.3.1 broke mosviz NIRISS parser

Ricky is on it, but documenting it here to let everyone know we are aware of this.

```
______________________________ test_niriss_parser ______________________________

mosviz_app = <jdaviz.configs.mosviz.helper.MosViz object at 0x7f4df2233f40>
tmpdir = local('/tmp/pytest-of-runner/pytest-0/test_niriss_parser0')

    @pytest.mark.remote_data
    def test_niriss_parser(mosviz_app, tmpdir):
    
        test_data = 'https://stsci.box.com/shared/static/l2azhcqd3tvzhybdlpx2j2qlutkaro3z.zip'
        fn = download_file(test_data, cache=True, timeout=30)
        with ZipFile(fn, 'r') as sample_data_zip:
            sample_data_zip.extractall(tmpdir)
    
        level3_path = (pathlib.Path(tmpdir) / 'NIRISS_for_parser_p0171')
    
        data_dir = level3_path
    
>       mosviz_app.load_niriss_data(data_dir)

../../.tox/py38-test-alldeps-cov/lib/python3.8/site-packages/jdaviz/configs/mosviz/tests/test_parsers.py:20: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../.tox/py38-test-alldeps-cov/lib/python3.8/site-packages/jdaviz/configs/mosviz/helper.py:198: in load_niriss_data
    super().load_data(data_obj, parser_reference="mosviz-niriss-parser")
../../.tox/py38-test-alldeps-cov/lib/python3.8/site-packages/jdaviz/core/helpers.py:41: in load_data
    self.app.load_data(data, parser_reference=parser_reference, **kwargs)
../../.tox/py38-test-alldeps-cov/lib/python3.8/site-packages/jdaviz/app.py:306: in load_data
    msg = parser(self, file_obj, **kwargs)
../../.tox/py38-test-alldeps-cov/lib/python3.8/site-packages/jdaviz/configs/mosviz/plugins/parsers.py:538: in mos_niriss_parser
    specs = SpectrumList.read(temp)
../../.tox/py38-test-alldeps-cov/lib/python3.8/site-packages/astropy/nddata/mixins/ndio.py:58: in __call__
    return registry.read(self._cls, *args, **kwargs)
../../.tox/py38-test-alldeps-cov/lib/python3.8/site-packages/astropy/io/registry.py:523: in read
    format = _get_valid_format(
../../.tox/py38-test-alldeps-cov/lib/python3.8/site-packages/astropy/io/registry.py:609: in _get_valid_format
    return _get_highest_priority_format(mode, cls, valid_formats)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

mode = 'read', cls = <class 'specutils.spectra.spectrum_list.SpectrumList'>
valid_formats = ['JWST x1d multi', 'JWST x1d MIRI MRS']

    def _get_highest_priority_format(mode, cls, valid_formats):
            ...
            elif priority > current_priority:
                best_formats = [format]
                current_priority = priority
    
        if len(best_formats) > 1:
>           raise IORegistryError("Format is ambiguous - options are: {}".format(
                ', '.join(sorted(valid_formats, key=itemgetter(0)))
            ))
E           astropy.io.registry.IORegistryError: Format is ambiguous - options are: JWST x1d multi, JWST x1d MIRI MRS

../../.tox/py38-test-alldeps-cov/lib/python3.8/site-packages/astropy/io/registry.py:643: IORegistryError
```

--- a/jdaviz/configs/mosviz/plugins/parsers.py
+++ b/jdaviz/configs/mosviz/plugins/parsers.py
@@ -535,7 +535,7 @@ def mos_niriss_parser(app, data_dir, obs_label=""):
                     else:
                         hdu.header["SRCTYPE"] = "EXTENDED"
 
-                specs = SpectrumList.read(temp)
+                specs = SpectrumList.read(temp, format="JWST x1d multi")
                 filter_name = [x
                                for x in fname.split("/")[-1].split("_")
                                if x[0] == "F" or x[0] == "f"][0]

